package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@adobe/node-httptransfer",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "Efficient File Transfer Module for NodeJS",
+  "author": {
+    "name": "Adobe Systems Inc."
```

Comment:
Please change to just "Adobe Inc". We were renamed to just `Adobe Inc.` about a year ago.

Reply:
Changed.